tox.ini

```diff
@@ -20,3 +20,6 @@ commands = python -m pytest {posargs}
 deps = flake8
 commands = flake8 sphinx_autodoc_typehints.py test_sphinx_autodoc_typehints.py
 skip_install = true
+
+[flake8]
```

Comment:
Any flake8 options go to setup.cfg. Also, this project uses a maximum line width of 99 columns, which is the maximum allowed by PEP 8. This is also obvious from setup.cfg. Therefore ignoring E501 is not acceptable. The E241 error is ignored in the default configuration (which is how flake8 have passed previously).